www/views/form.php

```diff
@@ -0,0 +1,351 @@
+
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet">
+<script src="https://cdn.jsdelivr.net/npm/vue@2"></script>
+
+<div id="app" class="bg-light">
+
+<div style=" background-color:#f0f0f0; padding-top:20px; padding-bottom:10px">
+  <div class="container" style="max-width:800px;">
+    <h3>Heat Pump Monitoring Submission: <?php if (isset($session["username"])) echo $session["username"]; ?></h3>
+    <p>If you have a heat pump and publish stats via emoncms, submit your details here.</p>
+  </div>
+</div>
+
+<div class="container" style="max-width:800px" >
+  <br>
+
+  <p><b>Email *</b></p>
+  <div class="input-group mb-3">
+    <input type="text" class="form-control" v-model="email" @change="update"> 
+  </div>
+  
+  <div class="row">
+    <div class="col">  
+      <p><b>Name *</b><br>Your name. This will not be displayed publicly but will allow for updating records if necessary.</p>
+      <div class="input-group mb-3">
+        <input type="text" class="form-control" v-model="name" @change="update"> 
+      </div>
+    </div>
+    <div class="col">  
+      <p><b>Vague Location *</b><br>Roughly where the heat pump is installed, to nearest city or county.</p>
+      <div class="input-group mb-3">
+        <input type="text" class="form-control" v-model="location" @change="update"> 
+      </div>
+    </div>
+  </div>
+  
+  <hr>
+  <h4>About Your Heating System</h4>
+  
+  <div class="row">
+    <div class="col">  
+      <p><b>Heat Pump Make / Model</b></p>
+      <div class="input-group mb-3">
+        <input type="text" class="form-control" v-model="hp_model" @change="update"> 
```

Comment:
This is one of the fields that needs editing by moderators. The alternative is to provide a list of all manufacturers and models, which will be a pain to maintain.

We'll keep text input for now.

Reply:
ok great